docs/release-playbook.md

```diff
@@ -28,6 +28,32 @@ Do these steps once per release.
          should be cherry-picked, any remaining issues should become release
          blockers.
 
+### Update the status of GitHub issues for incompatible changes
+
+In the below, _X.Y_ is a release you are cutting.
+
+#### Start new migration windows
+
+1. Search for all [open "incompatible-change" issues that have "migration-ready" labels](https://github.com/bazelbuild/bazel/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Aincompatible-change+label%3Amigration-ready)
+1. For each such issue:
+     1. Add a "migration-_X.Y_" label.
```

Comment:
Does this mean that issues authors should not set these labels? We may need more guidance to people (like me) when adding new incompatible feature flags.

Reply:
Yes, it does - it is all in the [policy](https://docs.google.com/document/d/1q5GGRxKrF_mnwtaPKI487P8OdDRh2nN7jX6U-FXnHL0/) but apparently this has not been widely enough read/followed, so I am trying to fix that.